suite-native/config/src/supportedNetworks.ts

```diff
@@ -48,10 +49,12 @@ export const discoverySupportedNetworks = [
     ...networkSymbolsWhitelistMap.testnet,
 ];
 
-export const sortNetworks = (networks: Network[]) =>
-    A.sort(networks, (a, b) => {
-        const aOrder = discoverySupportedNetworks.indexOf(a.symbol) ?? Number.MAX_SAFE_INTEGER;
-        const bOrder = discoverySupportedNetworks.indexOf(b.symbol) ?? Number.MAX_SAFE_INTEGER;
+export const orderedNetworksSymbols = Object.keys(networks) as NetworkSymbol[];
```

Comment:
I'd got with this name:
```suggestion
export const orderedNetworkSymbols = Object.keys(networks) as NetworkSymbol[];
```

Reply:
546048a009